### PR TITLE
Snapper without DBus

### PR DESCRIPTION
**Pull request by mnowaksuse**
Verification runs:
* `snapper create`: http://assam.suse.cz/tests/4931
* `snapper_undochange`: http://assam.suse.cz/tests/4936
* `snapper_cleanup`: http://assam.suse.cz/tests/5033
* `snapper_thin_lvm`: http://assam.suse.cz/tests/5057

### CODE REVIEW

File: lib/utils.pm [hunk 1]
```diff
@@ -711,6 +712,19 @@ sub handle_login {
     send_key "ret";
 }
 
+sub service_action {
```
Comment (okurz) [inline lib/utils.pm]: As we are having more and more util functions. Could you add some documentation comments? E.g. take testapi.pm as reference

File: tests/console/snapper_cleanup.pm [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
+# Summary: Snapper cleanup test based on FATE#312751
+# Maintainer: Dumitru Gutu <dgutu@suse.com>
 
 use base "consoletest";
 use strict;
-use utils;
 use testapi;
+use utils qw(service_action clear_console);
+
+our $snapper = 'snapper';
```
Comment (okurz) [inline tests/console/snapper_cleanup.pm]: why not define it as argument in the subs? looks safer to me

**Comment by okurz** [comment]
@dgutu-suse please take a look

**Comment by okurz** [comment]
+1 by me although it is getting too complicated for me to fully understand, I just have to trust you :-)

**Comment by mnowaksuse** [comment]
@okurz Please do :).

File: products/sle/main.pm [hunk 3]
```diff
@@ -601,6 +601,7 @@ sub load_consoletests() {
                 loadtest "console/installation_snapshots";
             }
             loadtest "console/snapper_undochange";
+            loadtest "console/snapper_create";
```
Comment (coolo) [inline products/sle/main.pm]: I take this out - as it breaks yast2_snapper in almost all scenarios.

Comment (mnowaksuse) [inline products/sle/main.pm]: Fixed & enabled in PR https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2453.

**Pull request merged**